en/components/general/cli-overview.md

```diff
@@ -1,70 +1,109 @@
+Install [`Ignite UI CLI`](https://github.com/IgniteUI/igniteui-cli) using the npm package manager:
 ```cmd
 npm install -g igniteui-cli 
 ```
-Once the  [`Ignite UI CLI`](https://github.com/IgniteUI/igniteui-cli) is installed, there are two possible options/modes to start the tool - guided experience and using specific commands.
+Once the installation process is finished, you can start the tool using either guided experience or specific commands.
 
 #### Using guided experience
-The shortest and easiest way to bootstrap an application is to use the Ignite UI CLI guided [`step by step mode`](https://github.com/IgniteUI/igniteui-cli/wiki/step-by-step), which creates a configured app that can be up and running with the ease of a single command. 
+The shortest and easiest way to bootstrap an application is to use the Ignite UI CLI [`Step-by-Step Guide`](cli/step-by-step-guide.md), which creates a configured app that can be up and running with the ease of a single command. 
 ```cmd
 ig
 ```
 
 <div style="display:inline-block;">
-    <a style="background: url(../../images/general/buildCLIapp.gif); display:flex; justify-content:center; min-width:540px; min-height:315px;"
+    <a style="background: url(../../../images/general/buildCLIapp.gif); display:flex; justify-content:center; min-width:540px; min-height:315px;"
        href="https://youtu.be/QK_NsdtdA70" target="_blank">
-        <img src="../../images/general/play.svg" style="vertical-align: middle;" />
+        <img src="../../../images/general/play.svg" style="vertical-align: middle;" />
     </a>
     <p style="text-align:center;">Building Your First Ignite UI CLI App</p>
 </div>
 
 #### Using commands
-We can always use the Ignite UI CLI [commands](#available-commands) for generating an Ignite UI project, adding a new component or building and serving the project by ourselves. For that purpose, we can use the following commands:
+To create new project, add a component template and build it use the following Ignite UI CLI [commands](#available-commands):
 ```cmd
 ig new <project name> --framework=angular --type=igx-ts 
 cd <project name>
 ig add <component/template> <component_name>
 ig start 
 ```
-- The full list of available **component/template** values is [here](https://github.com/IgniteUI/igniteui-cli/wiki/add#ignite-ui-for-angular-templates), you can run [`ig list`](#available-commands) command in your project directory to list all the available templates.
-- Both the **project name** and the **component_name** are custom values, provided by the developer.
-Here is an example of creating a new Ignite UI for Angular project with an igxGrid control:
+
+To execute the above commands, you should replace the *project name* and *component_name* placeholders with an actual value.
+After running these commands, your new project will be built, served and automatically opened in your default browser.
+
+### Scaffolding
```

Comment:
Yeah, nope. This entire new section is rather messy to read through (except maybe Add template) and basically the same content as under [Getting Started](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/getting_started.html#create-an-application-by-using-the-ignite-ui-cli).

This should be the more "in-depth" section. Some content belongs in the "Create project" topic, or since that topic is fairly small, it should be merged in here